app/src/test/java/ch/epfl/sdp/entity/PlayerTest.java

```diff
@@ -24,17 +21,7 @@ public void setup(){
         PlayerManager playerManager = new PlayerManager();
         player1 = new Player(6.149290, 46.212470, 50, "Skyris", "test@email.com");
         game.getInstance().setMapApi(new MockMapApi());
-        game.getInstance().setRenderer(new Renderer() {
-            @Override
-            public void display(Collection<Displayable> displayables) {
-
-            }
-
-            @Override
-            public void unDisplay(Displayable displayable) {
-
-            }
-        });
+        game.getInstance().setRenderer(new MockRenderer());
```

Comment:
I had removed MockRenderer for a reason, which was that there was way too many classes already. You will be really only using MockRenderer maximum 3 times, so having an abstraction for it is really overkill for its purpose. If you want to get rid of the ugly boilerplate when wiriting " new Renderer() { @Override ...{  }   @Override ...{  }} " you can just make MockMapApi implement Renderer and using empty definitions for the methods from Renderer. Try to use less classes whenever possible unless the new class really encapsulates a lot of functionality or state (or both)

Reply:
I combined MockLocationFinder, MockMapApi and MockRenderer into MockMap.